website_google_tag_manager/README.rst

```diff
@@ -1,6 +1,6 @@
-==================
```

Comment:
hmmm what has to deal this w/ no crawler? :thinking:

Reply:
@simahawk Nothings..
But because of that Travis failed.

Reply:
if you need this change to fix this build, pls, split it to a specific commit like `website_google_tag_manager: fix RST`